### PR TITLE
Change Heptio Ark references to Velero

### PR DESCRIPTION
**Pull request by nrb**
* Update CLI commands
* Use the volume-snapshot create command instead of YAML
* Update the provider name to portworx.io/portworx; this must match the
plugin's name in code.

Signed-off-by: Nolan Brubaker <brubakern@vmware.com>

### CODE REVIEW

**Comment by nrb** [comment]
@disrani-px Dinesh, will you be able to take a look at these docs? Thanks!

File: content/portworx-install-with-kubernetes/storage-operations/disaster-recovery/_index.md [hunk 1]
```diff
@@ -1,64 +1,48 @@
...
+{{<info>}}These instructions are for Velero v0.11 and higher. For older versions, please [click here.](ark-pre-0.10) {{</info>}}
 
-## Install Ark Plugin
+## Install Velero Plugin
 
-Run the following command to install the Portworx plugin for Ark:
+Run the following command to install the Portworx plugin for Velero:
 ```text
-ark plugin add portworx/ark-plugin:0.5
+velero plugin add portworx/ark-plugin:0.5
```
Comment (skriss) [inline content/portworx-install-with-kubernetes/storage-operations/disaster-recovery/_index.md]: hopefully we can update the container image as well

Comment (nrb) [reply content/portworx-install-with-kubernetes/storage-operations/disaster-recovery/_index.md]: Good call. I don’t know what that will be at the moment.

**Comment by disrani-px** [comment]
@nrb Thanks for the PR. We'll need to create a copy of this page for anyone using the older version of Ark.
Also, we are waiting for the new plugin to be qualified before updating the docs.

**Comment by disrani-px** [comment]
Fixed with #340

**Pull request closed without merging**